clip_object_tracker.py

```diff
@@ -24,6 +24,12 @@
 from deep_sort.tracker import Tracker
 from tools import generate_clip_detections as gdet
 
+from utils.yolov5 import Yolov5Engine
+
+classes = []
+
```

Comment:
should they put their class names here? Maybe we should have class names as a separate file. Better yet, the same data.yaml that they use in custom training.

Reply:
no. the class names are loaded form the model right now